diff --git a/configure.ac b/configure.ac
index 7d668276..81b3725f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -184,7 +184,7 @@ AS_IF([test "$with_protoc" != "no"], [
 ])
 AM_CONDITIONAL([USE_EXTERNAL_PROTOC], [test "$with_protoc" != "no"])
 
-AX_PTHREAD
+#AX_PTHREAD
 AM_CONDITIONAL([HAVE_PTHREAD], [test "x$ax_pthread_ok" = "xyes"])
 # We still keep this for improving pbconfig.h for unsupported platforms.
 AC_CXX_STL_HASH
